src/Servers/Kestrel/Core/src/Internal/Http/ZeroContentLengthMessageBody.cs

```diff
@@ -35,6 +35,8 @@ public override bool TryRead(out ReadResult result)
         }
 
         public override void Complete(Exception ex) { }
+        
+        public override Task CompleteAsync(Exception exception)=>Task.CompletedTask;
```

Comment:
I would just make this virtual instead of abstract and have the base version call Complete() and return Task.CompletedTask